### ISSUE TITLE
Add a scroll to the client profile

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw#1028]
The client profile is becoming quite long and some values are no longer visible on the device screen. We will need to add a scroll to the client profile to allow the user to view all part (top to bottom) 

@rowo Is there any design aspects we should consider for such a feature?

**Comment by rowo** [comment]
@msschroeder can you kindly share a screenshot of the problem?

**Comment by msschroeder** [comment]
![Screenshot_20191206-091735](https://user-images.githubusercontent.com/20777928/70301122-8cd91d00-180a-11ea-9dc4-36207aca71d2.png)

@roow Here is an example of when the profile is a family head or caregiver

**Comment by rowo** [comment]
Ok thanks @msschroeder. There are a few things to comment on here:
1. yes this profile view needs a scroll. Ideally it could work the same as the ANC app where when a user scrolls up, the top section gets squished down so only the client name appears in the section below the back button row.
2. I think I need to provide a better avatar with a colored background.
3. There's a lot of margin after MLEZI. This should be the same space as between the color break and the red overdue button.
4. The red overdue button has a lot of text. Is there a way to shorten this? 
5. The sections below also have a lot of text. Is there a way to shorten this?
6. The family due section should be left aligned.

**Comment by msschroeder** [comment]
@rowo Feedback on your comments:
1. _yes this profile view needs a scroll. Ideally it could work the same as the ANC app where when a user scrolls up, the top section gets squished down so only the client name appears in the section below the back button row._ - That would be fine for me
_2. I think I need to provide a better avatar with a colored background._ - Would it be possible to get a mock up or idea of what this would look like?
3. _There's a lot of margin after MLEZI. This should be the same space as between the color break and the red overdue button._ @manutarus What is the margin for a user profile when they are either the head of household or primary caregiver? It is quite large.
_4. The red overdue button has a lot of text. Is there a way to shorten this?
5.The sections below also have a lot of text. Is there a way to shorten this?_ -
The Swahili version is sometimes very long. And to not lose the meaning it is not always possible. The Jphiego team is aware of this, but it might forget as they are translating sometimes. We raised it as we were reviewing in September. However, they are continuously adding to Transifex. Also, it is not always easy to know where the text goes to and there are duplicates of the same text. So the only way to know what it relates to is once you see it. I guess the best way forward is I can talk to Jphiego and see if they can review them again and try and shorten them.

**Comment by rowo** [comment]
@msschroeder Thanks.
1. Cool, let's go with that. Let me know if I need to do anything.
2. Not too different. I think it's just fading into the background too much. See here:
  <img width="533" alt="Screen Shot 2020-01-05 at 10 34 07 PM" src="https://user-images.githubusercontent.com/1584163/71793624-89080880-300b-11ea-9899-4351eb385fa0.png">
4-5. Shorter text — gotcha. yeah I think here would be a situation to follow up with them. It should be useful, but this space doesn't let you communicate everything, so it's a balance about what needs to be highlighted if the straight translation is so long.

**Comment by manutarus** [comment]
@msschroeder @rowo Yes moving the profile to a collapsing toolbar layout would be nice, the margin after MLEZI gets used up if the member is a family head or/and primary caregiver but we can reduce this when not required.

**Comment by msschroeder** [comment]
@ellykits Is this done? Has it been merged?
